lib/iris/coords.py

```diff
@@ -32,6 +33,7 @@
     CellMeasureMetadata,
     CoordMetadata,
     DimCoordMetadata,
+    ConnectivityMetadata,
```

Comment:
@trexfeathers Could we make this alphabetical, thanks :+1:

Reply:
c2ac0fc05949f07cd3483e802994708aa427da22